tutorials/plot_metadata_epochs.py

```diff
@@ -69,35 +69,74 @@
 # of each epoch. We'll create a new column in our metadata object and use
 # it to generate averages for many subsets of trials.
 
-# Create a new metadata column
+# Create two new metadata columns
 meta = epochs.metadata
-is_concrete = meta["WordFrequency"] > meta["WordFrequency"].median()
+is_concrete = meta["Concreteness"] > meta["Concreteness"].median()
 meta["is_concrete"] = np.where(is_concrete, 'Concrete', 'Abstract')
+is_concrete = meta["NumberOfLetters"] > 5
+meta["is_long"] = np.where(is_concrete, 'Long', 'Short')
 epochs.metadata = meta
 
-# We'll create a dictionary so that we can plot with ``plot_compare_evokeds``
-categories = ["NumberOfLetters", "is_concrete"]
-avs = {}
-for (cat1, cat2), _ in epochs.metadata.groupby(categories):
-    query = 'NumberOfLetters == {} and is_concrete == "{}"'.format(cat1, cat2)
-    this_epochs = epochs[query]
-    avs["{}/{}".format(cat1, cat2)] = this_epochs.average()
+###############################################################################
+# Now we can quickly extract (and plot) subsets of the data. For example, to
+# look at words split by word length and concreteness:
+
+query = "is_long == '{0}' & is_concrete == '{1}'"
+evokeds = dict()
+for concreteness in ("Concrete", "Abstract"):
+    for length in ("Long", "Short"):
+        subset = epochs[query.format(length, concreteness)]
+        evokeds["/".join((concreteness, length))] = list(subset.iter_evoked())
 
-# Style the plot
-colors = np.linspace(0, 1, num=len(avs))
+# For the actual visualisation, we store a number of shared parameters.
 style_plot = dict(
-    colors=plt.get_cmap('YlGnBu_r')(colors),
-    linestyles={'Concrete': '-', 'Abstract': '--'}
+    colors={"Long": "Crimson", "Short": "Cornflowerblue"},
+    linestyles={"Concrete": "-", "Abstract": ":"},
+    split_legend=True,
+    ci=.68,
+    show_sensors=4,
+    show_legend=3,
```

Comment:
show_sensors -> sensors_loc
show_legend -> legend_loc

so I understand what these numbers are. Or at least I think i know. It would be even nicer to have:

legend_loc = 'lower left'
sensors_loc = 'upper right'

this should work.

Reply:
Yes, should have done it like this in the first place. The position numbers for MPL are weird.

Reply:
forget this for here. I see it has already been released.

Reply:
Don't forget by default it's simply a boolean switch for showing them or not showing them, so `sensors_loc` probably isn't intuitive.

Reply:
Maybe if we change kwarg to `legend` (or something like that) both `False` and `lower_left` would make sense.
But that was added in previous version, right?